check-licenses/action.yml

```
@@ -1,63 +1,53 @@
-name: "Check licenses"
-description: "Check if package licenses in the current environment are among the accepted ones"
+name: >
+  Check licenses action
+
+description: 
+  Verifies if the licenses of the dependencies installed in the current
+  environment are compliant with PyAnsys guidelines. This action is assumed to
+  used in its own job step. It clones the project and installs the project with
```

```suggestion
  be used in its own job step. It clones the project and installs the project with
```
Missing "be"? This edit might make the line length too long.